scripts/lochness_create_template.py

```diff
@@ -414,6 +419,8 @@ def create_config_template(config_loc: Path, args: object) -> None:
         base: /PHOENIX
 admins:
     - {args.email}
+
+sender: {args.email}
```

Comment:
Has this line been inadvertently pushed down?

Reply:
I did not realize that I've added the space. I'll remove it.

Reply:
f9ea86d

Reply:
> I did not realize that I've added the space.

That's not what I meant. This line was in 299 before now it is at 423. I am rather trying to understand if `sender` occurs at a different level in the config hierarchy than before.

Reply:
Yes I've moved it at the end of the config file for better visibility for myself